packages/xo-web/src/common/intl/messages.js

```diff
@@ -860,8 +860,7 @@ const messages = {
   srRescan: 'Rescan all disks',
   srReconnectAll: 'Connect to all hosts',
   srDisconnectAll: 'Disconnect from all hosts',
-  srForget: 'Forget this SR',
-  srsForget: 'Forget SRs',
+  srForget: 'Forget {nSrs, number} SR{nSrs, plural, one {} other {s}}?',
```

Comment:
This message is also used for the button's tooltip but the values are not passed there.